packages/openapi-code-generator/src/typescript/client/abstract-client-builder.ts

```diff
@@ -9,6 +9,10 @@ import {ClientOperationBuilder} from "./client-operation-builder"
 import {ClientServersBuilder} from "./client-servers-builder"
 
 export abstract class AbstractClientBuilder implements ICompilable {
+  protected abstract readonly capabilities: {
```

Comment:
Rudimentary capabilities concept to more easily allow for skew of feature parity between templates